### ISSUE TITLE
Bottom controls unreadable

### ISSUE THREAD

**Issue by gunslingerfry** [dough10/PolySonic#75]
![screenshot from 2016-10-11 12 33 00](https://cloud.githubusercontent.com/assets/4825657/19283592/b6ab3f1e-8faf-11e6-8dbc-843cbaf9b801.png)

Bottom controls are just a blurry mess.

Config: 
Chrome 53.0.2785.143
Ubuntu 14.04

**Comment by dough10** [comment]
yes this is fixed in the master here on Github. It just hasn't been pushed to the webstore. I will get a build made with this change and pushed to the webstore soon. I have changed jobs and moved recently so haven't put much time into the app.

**Comment by disanvparean** [comment]
Any news ?
It's a good app, but we can't use it anymore, that's sad :'(

**Comment by dough10** [comment]
I honestly forgot about this.   I Haven't been using subsonic for quite sometime.  If I remember correctly I just need to make a build and push to web store.  I'll get off my ass and see about posting it.

**Comment by dough10** [comment]
I have looked @ this in chrome 58.0 and it looks like the project is dead. attempting to open the app with old configs gives endless errors in console.  Attempting to create a config doesn't work either cause the animation for drop downs is broken. I am afraid there isn't much i can do without a complete rewrite of the app.

**Comment by disanvparean** [comment]
Hello, firstly thank you for your interest and your work.
That's a bit surprising as I'm using it in Iridium 58 (= Chrome 58). I just installed it yesterday and everything is working except the blurry bottom controls.

**Comment by dough10** [comment]
I think the issue is more attempting to build the app on my pc. I have deleted all the original files I used last when making a build. I am now having to attempt rebuild from source and some of the files are no longer available.  I used Polymer 0.5 to create this app and it has long been abandoned.
